docs/api/en/materials/MeshPhysicalMaterial.html

```diff
@@ -188,7 +188,7 @@ <h3>[property:Float transmission]</h3>
 
 		The transmission property can be used to model these materials.<br />
 
-		When transmission is non-zero, [page:Material.opacity opacity] should be set to *1*.
+		When transmission is non-zero, [page:Material.opacity opacity] should be set to *1*. This feature can only be used with a WebGL 2 rendering context.
```

Comment:
These two sentences are unrelated. For clarity, I'd put the WebGL 2 limitation on a separate line.

Reply:
Makes sense! Let me update the PR.